core/src/main/java/org/apache/accumulo/core/client/ClientConfiguration.java

```diff
@@ -199,15 +199,23 @@ public ClientConfiguration(Configuration... configs) {
   }
 
   /**
-   * Attempts to load a configuration file from the system. Uses the "ACCUMULO_CLIENT_CONF_PATH" environment variable, split on File.pathSeparator, for a list
-   * of target files. If not set, uses the following in this order- ~/.accumulo/config $ACCUMULO_CONF_DIR/client.conf -OR- $ACCUMULO_HOME/conf/client.conf
-   * (depending on whether $ACCUMULO_CONF_DIR is set) /etc/accumulo/client.conf
+   * Attempts to load a configuration file from the system using the default search paths. Uses the <em>ACCUMULO_CLIENT_CONF_PATH</em> environment variable,
+   * split on <em>File.pathSeparator</em>, for a list of target files.
+   * <p>
+   * If <em>ACCUMULO_CLIENT_CONF_PATH</em> is not set, uses the following in this order:
+   * <ul>
+   * <li>~/.accumulo/config
+   * <li>either:
+   * <ul>
+   * <li><em>$ACCUMULO_CONF_DIR</em>/client.conf, if <em>$ACCUMULO_CONF_DIR</em> is defined.
+   * <li><em>$ACCUMULO_HOME</em>/conf/client.conf, otherwise.
+   * </ul>
+   * <li>/etc/accumulo/client.conf
+   * <li>/etc/accumulo/conf/client.conf
+   * </ul>
+   * <p>
    *
-   * A client configuration will then be read from each location using PropertiesConfiguration to construct a configuration. That means the latest item will be
-   * the one in the configuration.
-   *
-   * @see PropertiesConfiguration
-   * @see File#pathSeparator
+   * @since 1.6.0
```

Comment:
This since tag is not needed because its the same as the class level since tag.  Methods inherit the since tag from the class.   Only need to set it on method if it differs from class since tag.

Reply:
Thanks for the catch! I realized I had inadvertently removed a line of documentation when I was performing the previous revision.